### ISSUE TITLE
(v9) Badge: insufficient color contrasts

### ISSUE THREAD

**Issue by tringakrasniqi** [microsoft/fluentui#21126]
### Environment Information

- **Package version(s)**: v9 beta
- **Browser and OS versions**: Edge

### Describe the issue:

Using the FluentUI **dark theme**:
- Badge with the prop `apperance="ghost"` and `appreance="outline"` do not fulfill the minimum contrast ratio requirements. The ratio currently is 2.76:1, the below picture shows the test done using the color contrast analyzer:
![dark-theme-visual-test-2px](https://user-images.githubusercontent.com/12565826/147578373-38d6ddb2-0b1b-429d-aba7-f7e16873cbd8.png)

- Badge with the following props ` appearance="filled" color="warning"` does not meet the min contrast ratio:
![dark_mode_filled_yellow_warning](https://user-images.githubusercontent.com/12565826/147578721-2c857649-9099-4821-80b8-d1f1d2c3ddf5.png)
Current ratio is: 1.29:1

Using the FluentUI **light theme**:
- Badge with props `appearance="outline" color="informative"` and `appearance="outline" color="subtle"` are almost invisible:
![light_mode_badge](https://user-images.githubusercontent.com/12565826/147579427-f5fd45fd-77a5-4562-a031-a817ed57b49e.png)

- PresenceBadge: the color of the status `away` has contrast ratio of 2.15:1 (as it is non-text visual needs a 3:1 ratio).
![small-contrast-wcag-aa-3-1 ratio](https://user-images.githubusercontent.com/12565826/147579777-5c90a294-40c6-4d9f-a322-3ad22cf2fa9e.png)

### Please provide a reproduction of the issue in a codepen:
Codesandbox: https://codesandbox.io/s/badge-a11y-review-bhu70

#### Expected behavior:

- Next should meet contrast ratio of 4:5:1, large text and non-textual elements 3:1.

**Comment by tringakrasniqi** [comment]
Update: this is still waiting response from the design team to provide different color codes and design tokens. 

I'm unassigning this until there is some other update.
